duckplayer/duckplayer-api/src/main/java/com/duckduckgo/duckplayer/api/DuckPlayer.kt

```diff
@@ -128,7 +128,7 @@ interface DuckPlayer {
      * @param uri The URI to check.
      * @return True if the URI is a YouTube no-cookie URI, false otherwise.
      */
-    suspend fun isSimulatedYoutubeNoCookie(uri: Uri): Boolean
+    fun isSimulatedYoutubeNoCookie(uri: Uri): Boolean
```

Comment:
There's also `suspend fun isSimulatedYoutubeNoCookie(uri: String): Boolean` should that one remove suspend too? Do we need two methods with different signatures?

Reply:
Not really, I've removed it